README.md

```diff
@@ -17,6 +18,12 @@ imgix is a real-time image processing service and CDN. It lets you edit images o
   * [URL Reconstruction](#url-reconstruction)
   * [What is the `ixlib` param?](#what-is-the-ixlib-param)
 
+<a name="swift-version"></a>
```

Comment:
Instead of adding this as a whole new section, can we put these notes under the existing "Usage > Swift" section?